firmware/Core/Src/antenna_deploy_drivers/ant_commands.c

```diff
@@ -294,13 +294,14 @@ uint8_t ANT_CMD_report_antenna_deployment_activation_count(enum ANT_i2c_bus_mcu
     return status;
 }
 
-/// @brief writes the cumulative time (in 50ms increments) that the deployment system has been active (for a specified antenna and mcu) in a response buffer.
+/// @brief writes the cumulative time (in ms increments) that the deployment system has been active (for a specified antenna and mcu) in a response buffer.
 /// @param i2c_bus_mcu specifies which mcu on the antenna deployment system to transmit to, and which i2c bus to use
 /// @param antenna the antenna to check. A number between 1-4
-/// @param response a 2 byte buffer where the cumulative deployment time (in 50ms increments) will be written. divide the response by 20 to get seconds.
+/// @param result a 2 byte buffer where the cumulative deployment time (in ms increments) will be written. divide the response by 20 to get seconds.
 /// @return 0 when the antenna deployment system has received the command, >0 otherwise
 /// @note data written to the result buffer is only valid if 0 was returned. One should check this before using the result.
-uint8_t ANT_CMD_report_antenna_deployment_activation_time(enum ANT_i2c_bus_mcu i2c_bus_mcu, uint8_t antenna, uint16_t *result) {
+//TODO in search, find all instances of this function and change result to uint32_t REMEMBER THIS!!!!!!
+uint8_t ANT_CMD_report_antenna_deployment_activation_time(enum ANT_i2c_bus_mcu i2c_bus_mcu, uint8_t antenna, uint32_t *result) {
```

Comment:
Recommendation: rename from `_report_` to `_get_`

Reply:
Got it! got rid of that todo function.